### DISCUSSION TITLE
question about monitoring enabling

### DISCUSSION THREAD

**Discussion by gaord** [rook/rook#12566] [General]
I would like to set up monitoring stack for ceph@rook 1.10.13. I do it by the document here:https://rook.io/docs/rook/v1.10/Storage-Configuration/Monitoring/ceph-monitoring/
I also enabled the monitoring setting in cephcluster object. However, it look like no targets are configured in prometheus.

Anything missing in my steps here?

Thanks.

**Comment by reefland** [comment]
My setup I only see one target, `serviceMonitor/rook-ceph/rook-ceph-mgr/0 (1/1 up)`.

```
$ k get servicemonitor -n rook-ceph

NAME            AGE
rook-ceph-mgr   251d
```

You have a serviceMonitor?
